.github/workflows/test.yml

```diff
@@ -42,6 +42,10 @@ jobs:
             python:
               version: "3.10"
               toxenv: py310,smoke
+          - os: ubuntu-latest
+            python:
+              version: '3.11.0-alpha - 3.11' # SemVer's version range syntax
+              toxenv: py311,smoke
```

Comment:
Since it already works fine, can we have this above with the normal matrix entries as it's also on ubuntu?